### DISCUSSION TITLE
How can users override the privacy level for fields ?

### DISCUSSION THREAD

**Discussion by coolbung** [Sunbird-RC/community#226] [General]
Currently, field level privacy can be configured at the schema level - https://docs.sunbirdrc.dev/use/schema-configuration#visibility-scope

How can this be set so that the user can override the privacy for the field i.e. if there's an address field which is set to Private the schema, but a user wishes to make it Public for his profile. 

@dileepbapat @tejash-jl 

cc @parthlawate @pawana

**Comment by dileepbapat** [comment]
@coolbung This **user** preference for the visibility scope was discussed but not implemented. Currently registry level schema / policy is supported. We might need to spec out in detail if this feature needs to be added.
